py/path/local.pyi

```diff
@@ -0,0 +1,94 @@
+import sys
```

Comment:
For `py.path`, we should only expose `class local`. `LocalPath` is not importable directly, only as the `py.path.local`. And the SVN stuff I guess is dead and better to leave out.

Should remove everything else from this file and `py/path/common.pyi` or expose underscored if referenced indirectly by an exposed item.